### PR TITLE
Chore/cuda chore

### PR DESCRIPTION
**Pull request by agnesLeroy**
### Description

This PR introduces formatting on cuda sources in the CI.
The command
```
cargo xtask chore_format
```
now formats cuda sources as well as the Rust sources.
Some variables are also renamed in this PR to match new notations in Concrete-core.

### Checklist 

(Use '[x]' to check the checkboxes, or submit the PR and then click the checkboxes)

* [x] Tests for the changes have been added (for bug fixes / features)
* [x] Docs have been added / updated (for bug fixes / features)
* [x] The PR description links to the related issue (to link an issue, use '#XXX'.)
* [x] The tests on AWS have been launched and are successful (comment with @slab-ci cpu_test and/or @slab-ci gpu_test to trigger the tests)
* [x] The draft release description has been updated
* [x] Check for breaking changes (including serialization changes) and add them to commit message following the conventional commit [specification][conventional-breaking]

[conventional-breaking]: https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with-description-and-breaking-change-footer

### CODE REVIEW

**Comment by agnesLeroy** [comment]
@pdroalves @bbarbakadze I need to add some more renaming in this one, I'll notify you when it's ready for review.

**Comment by agnesLeroy** [comment]
@slab-ci gpu_test

**Comment by agnesLeroy** [comment]
@slab-ci gpu_test

**Comment by agnesLeroy** [comment]
@slab-ci gpu_test

**Comment by agnesLeroy** [comment]
@slab-ci gpu_test

File: concrete-cuda/cuda/include/bootstrap.h [hunk 1]
```diff
@@ -8,171 +8,92 @@ extern "C" {
...
+    uint32_t lwe_dimension_after, uint32_t glwe_dimension,
+    uint32_t base_log_bsk, uint32_t level_count_bsk, uint32_t base_log_ksk,
+    uint32_t level_count_ksk, uint32_t number_of_samples);
 };
 
 #ifdef __CUDACC__
 __device__ inline int get_start_ith_ggsw(int i, uint32_t polynomial_size,
                                          int glwe_dimension,
-                                         uint32_t l_gadget);
+                                         uint32_t level_count);
```
Comment (pdroalves) [inline concrete-cuda/cuda/include/bootstrap.h:86]: Don't we use "decomp_level" in other places?

File: concrete-cuda/cuda/include/keyswitch.h [hunk 2]
```diff
@@ -5,20 +5,15 @@
...
-
-void cuda_keyswitch_lwe_ciphertext_vector_64(void *v_stream, void *lwe_out, void *lwe_in,
-                        void *ksk,
-                        uint32_t lwe_dimension_before,
-                        uint32_t lwe_dimension_after,
-                        uint32_t base_log, uint32_t l_gadget,
-                        uint32_t num_samples);
+void cuda_keyswitch_lwe_ciphertext_vector_32(
+    void *v_stream, void *lwe_array_out, void *lwe_array_in, void *ksk,
+    uint32_t lwe_dimension_before, uint32_t lwe_dimension_after,
```
Comment (pdroalves) [inline concrete-cuda/cuda/include/keyswitch.h:10]: Is there a reason to call these variables "before" and "after"? Maybe we could change them to "out" and "in" to match lwe_array_out and lwe_array_in.

**Review by pdroalves — APPROVED** [review]
Hey @agnesLeroy !
Thanks for this PR. It will be awesome to have linting working in concrete-cuda.

Everything seems good to me. I just added some comments about minor things that caught my attention, but we are good to go if you wish.

File: concrete-cuda/cuda/include/bootstrap.h (hunk 1 above)
Comment (agnesLeroy) [reply concrete-cuda/cuda/include/bootstrap.h:86]: decomp_level corresponds to a level index, while level_count corresponds to the total number of levels if I'm correct. I could rename decomp_level into level to make it simpler.

File: concrete-cuda/cuda/include/keyswitch.h (hunk 2 above)
Comment (agnesLeroy) [reply concrete-cuda/cuda/include/keyswitch.h:10]: The before and after wording is inherited from Concrete-core but actually we're dropping these notations there, so yes naming them lwe_dimension_in and lwe_dimension_out makes sense :slightly_smiling_face: I'll do that

File: concrete-cuda/cuda/include/bootstrap.h (hunk 1 above)
Comment (agnesLeroy) [reply concrete-cuda/cuda/include/bootstrap.h:86]: Renamed locally

**Comment by agnesLeroy** [comment]
@slab-ci gpu_test

**Comment by agnesLeroy** [comment]
@slab-ci cpu_test

**Pull request merged**